karaf/features/src/main/feature/feature.xml

```diff
@@ -16,12 +16,17 @@
     See the License for the specific language governing permissions and
     limitations under the License.
 -->
-<features xmlns="http://karaf.apache.org/xmlns/features/v1.2.0" name="org.apache.brooklyn-${project.version}">
+<features xmlns="http://karaf.apache.org/xmlns/features/v1.2.0"
+          name="org.apache.brooklyn-${project.version}"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:noNamespaceSchemaLocation="http://karaf.apache.org/xmlns/features/v1.2.0">
 
     <repository>mvn:org.apache.karaf.features/standard/${karaf.version}/xml/features</repository>
     <repository>mvn:org.apache.karaf.features/enterprise/${karaf.version}/xml/features</repository>
     <repository>mvn:org.apache.karaf.features/spring/${karaf.version}/xml/features</repository>
 
+    <!-- TODO how to add central.maven.org repo? -->
+    <repository>http://central.maven.org/maven2/org/apache/jclouds/karaf/jclouds-karaf/1.9.2/jclouds-karaf-1.9.2-features.xml</repository>
```

Comment:
Yes, that's the one. Unfortunately it's not able to find the jar file when using the `mvn` protocol.
Looking at the debug output it's trying to download from:
  * http://svn.apache.org/repos/asf/servicemix/m2-repo/
  * https://repo.maven.apache.org/maven2/

It's probably a bug that it's not in the apache repo, but still it got to the `central.maven.org` repo. Any way to add repos to search in like we can do in maven?